src/SolutionTemplate/UnoSolutionTemplate.net6/Mobile/UnoQuickStart.Mobile.csproj

```diff
@@ -9,8 +9,6 @@
 		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">maccatalyst-x64</RuntimeIdentifier>
 		<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-macos'">osx-x64</RuntimeIdentifier>
```

Comment:
I suppose there must be `ios-arm64` identifier with some condition in order to make `dotnet publish` work

Reply:
Yes, it's best to provide it explicitly using `-f`

Reply:
The issue is that restore does not happen for `ios-arm64`
How to fix that?